### ISSUE TITLE
Decide on schematic and data curator repos structure

### ISSUE THREAD

**Issue by milen-sage** [Sage-Bionetworks/schematic#272]
@xdoan and @sujaypatil96  since we brought structuring forks/branches in slack, @allaway and @xindiguo have had questions, suggestions and ideas regarding how to go about this.

Slack is bad for getting chat history so hopefully here we'll get all in the same conversation.

The main issue is github no longer supports forking a repository in the same organization under the same account (even if the fork is a different name); see
- https://github.community/t/alternatives-to-forking-into-the-same-account/10200
- and related discussion here https://stackoverflow.com/questions/22767617/copy-fork-a-git-repo-on-github-into-same-organization

The standard workarounds suggested in the github documentation may be appealing. But has anyone worked with a different setup?

For HTAN, we have an external github organization: nci-htan; that's where the HTAN-specific forks live both for schematic and the data curator app. The goal is to have the Sage Bionetworks organization just contain the "generic/core" codebases. Are there github organizations that can host the NF, Fred Hutch Accelerator and other upcoming projects? Is this something that Sage has bumped into before and there are existing guidelines?

@allaway if we decide that instead of forks we work with completely different repos 
- is PR-ing changes back into the core upstream repo sufficiently straightforward
- is pulling and merging latest code sufficiently straightforward
- is having consistent commit history (which is nicely captured by forks) important; if it is can it be still captured if we are using different repos on github (sorry I haven't worked with this setup before and I am not sure what's the behavior)

**Comment by allaway** [comment]
>Are there github organizations that can host the NF, Fred Hutch Accelerator and other upcoming projects?  
No, but we could create one. 

>is PR-ing changes back into the core upstream repo sufficiently straightforward
>is pulling and merging latest code sufficiently straightforward

These first two are _possible_ but not super straightforward (at least for me, others may think it's easy). It requires some extra work on the command line, and (i think) is pretty easy to get stuck or confused with multiple remotes. 

>is having consistent commit history (which is nicely captured by forks) important; if it is can it be still captured if we are using different repos on github (sorry I haven't worked with this setup before and I am not sure what's the behavior)

I'm not sure if it's important, but I think that diverging commit histories could get hairy quickly...

All in all, I think that the three questions you raise are enough of an argument to just create a new org (for NF) even if it's just for this one fork...it is free, after all. It might be nice to make Sage IT (cc @ahayden) aware of these satellite orgs, if/when they are created, and perhaps add someone from IT as an admin to them, if that's necessary.

**Comment by milen-sage** [comment]
Good point - I'll talk to James and/or Aaron to see how do we keep track of proliferating code; perhaps we shouldn't be too worried as long as the code that ends up in the Sage repos is peer-reviewed by someone at Sage.
